js/main.js

```diff
@@ -0,0 +1,42 @@
+$(function(){
+// document ready function
+
+    $("#enter-city").click(updateCity);
```

Comment:
@eloisebarrow you could also used `.on('click', updateCity)`. It is more peformant and also allows you t attach more than event on the listener. For example  `.on('click', 'mouseover', updateCity)`